openkfw/TruBudget
# 2028: Restrict max length of title/subtitle for project/subproject

### Checklist




### Description
- see #2028 

### How to test

1. login as any user
2. Try to create new project/subproject
3. Test if you can or cannot enter title/subtitle longer then 35 characters
4. It should not be possible

Closes #2028
